package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "lecture-08-node-api",
+  "version": "0.0.1",
+  "description": "lab 08 gnoevil",
+  "main": "server.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
From now on, please put a test script that connects to the Mocha folder and a start script that connects to your nodemon folder so that I can just type ``npm run test`` to run your tests and I can type ``npm run start`` so I can turn on the server using nodemon and I don't have to manually shut down/turn on my server if I make changes. :)